src/Plot.js

```diff
@@ -53,6 +53,24 @@ export default class Plot extends Viz {
           fontResize: true
         }
       },
+      ariaLabel: (d, i) => {
+        let ariaLabelStr = "";
+        if (d.nested) {
+          ariaLabelStr = `${this._drawLabel(d.data, d.i)}`;
+          if (d.data.x !== undefined) ariaLabelStr += `, x: ${d.data.x}`;
+          if (d.data.y !== undefined) ariaLabelStr += `, y: ${d.data.y}`;
+          if (d.data.x2 !== undefined) ariaLabelStr += `, x2: ${d.data.x2}`;
+          if (d.data.y2 !== undefined) ariaLabelStr += `, y2: ${d.data.y2}`;
+        }
+        else {
+          ariaLabelStr = `${this._drawLabel(d, i)}`;
+          if (d.x !== undefined) ariaLabelStr += `, x: ${d.x}`;
+          if (d.y !== undefined) ariaLabelStr += `, y: ${d.y}`;
+          if (d.x2 !== undefined) ariaLabelStr += `, x2: ${d.x2}`;
+          if (d.y2 !== undefined) ariaLabelStr += `, y2: ${d.y2}`;
+        }
+        return `${ariaLabelStr}.`;
+      },
```

Comment:
We can't be directly accessing each value as `d.x`, `d.y`, etc, because the user may have set `x("export_val")`, and in that case we'll need to use `this._x(d, i)`.

Additionally, after looking into this further, it doesn't make sense for the nested labels to have x/y values, as there will be multiple values for a given shape. If the shape is nested, just return the label string.